src/Loading.svelte

```diff
@@ -136,18 +148,20 @@
   }
 </style>
 
-<div class="container">
-  <div
-    class="spinner"
-    class:fade-in={fadeIn}
-    class:small
-    class:center
-    class:margins
-    class:condensed>
-    <div class="bounce1" style="background-color: var(--color-{color})" />
-    {#if !condensed}
-      <div class="bounce2" style="background-color: var(--color-{color})" />
-      <div class="bounce3" style="background-color: var(--color-{color})" />
-    {/if}
+{#if show}
+  <div class="container">
+    <div
+      class="spinner"
+      class:fade-in={fadeIn}
+      class:small
+      class:center
+      class:margins
+      class:condensed>
+      <div class="bounce1" style="background-color: var(--color-{color})" />
+      {#if !condensed}
+        <div class="bounce2" style="background-color: var(--color-{color})" />
+        <div class="bounce3" style="background-color: var(--color-{color})" />
+      {/if}
+    </div>
   </div>
-</div>
+{/if}
```

Comment:
Can we do this with a class instead? `class:show`?

Reply:
It's doable, but why would we? Wrapping it in `{#if}` prevents the DOM elements to be rendered, making it less performant and also less _idiomatic_ Svelte.

```
diff --git a/src/Loading.svelte b/src/Loading.svelte
index 73957a3..6166af4 100644
--- a/src/Loading.svelte
+++ b/src/Loading.svelte
@@ -21,7 +21,7 @@

 <style>
   .container {
-    display: flex;
+    display: none;
     align-items: center;
     justify-content: center;
     height: 100%;
@@ -146,22 +146,24 @@
       opacity: 1;
     }
   }
+
+  .show {
+    display: flex;
+  }
 </style>

-{#if show}
-  <div class="container">
-    <div
-      class="spinner"
-      class:fade-in={fadeIn}
-      class:small
-      class:center
-      class:margins
-      class:condensed>
-      <div class="bounce1" style="background-color: var(--color-{color})" />
-      {#if !condensed}
-        <div class="bounce2" style="background-color: var(--color-{color})" />
-        <div class="bounce3" style="background-color: var(--color-{color})" />
-      {/if}
-    </div>
+<div class:show class="container">
+  <div
+    class="spinner"
+    class:fade-in={fadeIn}
+    class:small
+    class:center
+    class:margins
+    class:condensed>
+    <div class="bounce1" style="background-color: var(--color-{color})" />
+    {#if !condensed}
+      <div class="bounce2" style="background-color: var(--color-{color})" />
+      <div class="bounce3" style="background-color: var(--color-{color})" />
+    {/if}
   </div>
-{/if}
+</div>
```

Reply:
Cool let's leave it, I think for a loading indicator it's fine as-is.